docker-compose.yml

```diff
@@ -1,37 +1,38 @@
-# docker-compose.yml
 version: '3'
 networks:
   todoapp-net:
 services:
   pythonapp:
-    image: todoapp:2.0.0
+    image: volry/todoapp:1.0.0
     build:
       context: .
       dockerfile: Dockerfile
     ports:
       - "8080:8080"
     environment:
       - PYTHONUNBUFFERED=1
+      - DB_ENGINE=${DB_ENGINE}
+      - DB_NAME=${DB_NAME}
+      - DB_USER=${DB_USER}
+      - DB_PASSWORD=${DB_PASSWORD}
+      - DB_HOST=${DB_HOST}
+      - DB_PORT=${DB_PORT}
     networks:
       - todoapp-net
     depends_on:
       - mysql
     restart: unless-stopped
   mysql:
-    image: mysql-local:1.0.0
-    build:
-      context: .
-      dockerfile: Dockerfile.mysql
-    container_name: my-sql
+    image: mysql:5.7
```

Comment:
Why are you using a base image instead of an image built from the Dockerfile.mysql file?